### ISSUE TITLE
Insufficient filename widget spacing in Portuguese (Portugal)

### ISSUE THREAD

**Issue by eloquence** [freedomofpress/securedrop-client#1535]
## Description

When the language is set to Portuguese, there is insufficient spacing between the "print" action and the filename

## Steps to Reproduce

1. Start the app with changes in #1531 using `LANG=pt_PT securedrop-client` or similar.

## Expected Behavior

All widgets the user can interact with are clearly differentiable through sufficient spacing between them.

## Actual Behavior

There is no spacing between the print (`IMPRIMIR`) action and the filename, making them not clearly differentiable.

![pt-spacing](https://user-images.githubusercontent.com/213636/178627287-0e41920a-c756-4210-9e74-d3652c1a1974.png)

**Comment by gonzalo-bulnes** [comment]
I'd suggest looking at this with a more systematic approach. Whatever we call that "file widget", it's components should be laid out in a way that allows their contents to vary while retaining usability (including readability in this case). I'm sure we can find way of laying out the widget so that the spacing between its components does not depend on minor variations of their lengths.

I'll have this one in mind when we got to work on the conversation widget.

**Comment by tina-ux** [comment]
I agree with you Gonzalo. I am not familiar with Qt (yet) but would expect a standard padding / spacing between components regardless of component length, same logic as "margin" for css. If you have a chance to fix it, you have my two thumbs up.
